backend/models.py

```diff
@@ -0,0 +1,54 @@
+from sqlalchemy import Column
+from sqlalchemy.types import Integer
+from sqlalchemy.types import String
+from sqlalchemy.types import Boolean
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+class User(db.Model):
+    """
+    User model - for admin
+    """
+    __tablename__ = 'user'
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    username = Column(String(20), unique=True)
+    hashed_password = Column(String(100))
+
+    def get_id(self):
+        return str(self.id)
+
+    def get(self):
+        return self
+
+
+class Participant(db.Model):
+    """
+    Participant model
+    """
+    __tablename__ = 'participant'
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    name = Column(String(20))
+    lastname = Column(String(20))
+    email = Column(String(30), default="")
+    github = Column(String(200), default="")
+    phone = Column(Integer, default=0 )
```

Comment:
it should be nullable, not set default to 0. And maybe it should be a string rather then integer?